Update cloud configuration pages

mjang opened the pull request:
Note the `<MUST FIX>` entry. We're offering a new feature, but have yet to define how to contact Qdrant for more info

Changes for _NEW_ cloud configuration options, as defined by @Rendez . Do not merge until we're ready to release this new feature.

Our docs are _not_ up to date with our current Cloud UI. I've also made corrections in this PR.

This PR affects three pages. You can review the build of these pages here:

- https://deploy-preview-595--condescending-goldwasser-91acf0.netlify.app/documentation/cloud/
- https://deploy-preview-595--condescending-goldwasser-91acf0.netlify.app/documentation/cloud/quickstart-cloud/
- https://deploy-preview-595--condescending-goldwasser-91acf0.netlify.app/documentation/cloud/create-cluster/

mjang commented on qdrant-landing/content/documentation/cloud/create-cluster.md:
```diff
@@ -10,24 +10,47 @@ This page shows you how to use the Qdrant Cloud Console to create a custom Qdran
+- Backup and disaster recovery
+- Horizontal and vertical scaling
+- Monitoring and log management
+
```
```suggestion

To learn more about this paid feature, contact us at <FILL IN BLANK>.

```

Rendez commented on qdrant-landing/content/documentation/cloud/create-cluster.md:
```diff
@@ -10,24 +10,49 @@ This page shows you how to use the Qdrant Cloud Console to create a custom Qdran
+1. Choose the number of vCPUs per node (0.5 core to 16 cores). If you add more RAM, you see options for
+   more CPUs.
+1. Select the number of nodes you want the cluster to be deployed on. 
 > Each node is automatically attached with a disk space offering enough space for your data if you decide to put the metadata or even the index on the disk storage.
```
Nowhere is mentioned that you can increase disk capacity (additional disk/extra disk), is this intentional?

mjang replied:
Thanks for catching, see this [new line](https://github.com/qdrant/landing_page/pull/595/files#diff-8dc2c0982a232200859b7981d9a1e638fd283ccbf3f12486b82f702f94531b25R31)

Rendez commented on qdrant-landing/content/documentation/cloud/create-cluster.md:
```diff
@@ -10,24 +10,49 @@ This page shows you how to use the Qdrant Cloud Console to create a custom Qdran
+1. Select **Clusters** and then click **+ Create**.
+1. In the **Create a cluster** screen select **Free** or **Standard**
+   For more information on a free cluster, see the [Cloud quickstart](/documentation/cloud/quickstart-cloud). The remaining steps assume you want a standard cluster.
+1. Select a provider. Currently, you can deploy to AWS, GCP, or Azure. 
```
Should provider names be in full? Amazon Web Services (AWS), Google Cloud Platform (GCP)?

And also, should they be within href _blank?

mjang replied:
It's a good idea to have provider names in full, once per page, so I've added them [here](https://github.com/qdrant/landing_page/pull/595/files#diff-8dc2c0982a232200859b7981d9a1e638fd283ccbf3f12486b82f702f94531b25R18-R20).

mjang commented:
@Rendez , I've updated this PR. I've also updated the screenshot in the [Create a cluster](https://qdrant.tech/documentation/cloud/create-cluster/) page

For reasons unknown to me, this PR isn't building updated docs. The current status for Netlify suggests that everything is OK. (I doubt it :) )

mjang commented on qdrant-landing/content/documentation/cloud/create-cluster.md:
```diff
@@ -10,24 +10,58 @@ This page shows you how to use the Qdrant Cloud Console to create a custom Qdran
+- Horizontal and vertical scaling
+- Monitoring and log management
+
+To learn more about this paid feature, contact us at <FILL IN BLANK>.
```
`<MUST FIX>` I've asked appropriate people what to put in here.

mjang replied:
Updated to 

`Learn more about these features in the [Qdrant Cloud dashboard](https://cloud.qdrant.io/).

mjang commented on qdrant-landing/content/documentation/cloud/create-cluster.md:
```diff
@@ -10,24 +10,58 @@ This page shows you how to use the Qdrant Cloud Console to create a custom Qdran
+| Disk space | 2 TB        |
+| Nodes      | 10          |  
+
+### Included features
+
+The features included with this paid cluster are:
```
```suggestion
### Included features (paid)

The features included with this cluster are:
```

Rendez reviewed (approved):
Depends on https://github.com/qdrant/qdrant-cloud-ui/pull/744 and on enabling the cluster creation page, which shall be done next week (1st week of March)

mjang commented:
Thanks @Rendez for telling me that https://github.com/qdrant/qdrant-cloud-ui/pull/744 was merged. Since we no longer have a dependency, I'm merging this doc change.
